### ISSUE TITLE
Crash on export array with undefined/null values

### ISSUE THREAD

**Issue by bas-vk** [robertkrimen/otto#165]
Example:
```
vm := otto.New()
vm.Run(`var arr = [undefined, null]`)
arr, _ := vm.Get("arr")
arr.Export()
```

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0xa0 pc=0x529400]

goroutine 1 [running]:
panic(0x6c3720, 0xc82000a150)
	/usr/lib/go/src/runtime/panic.go:464 +0x3e6
github.com/robertkrimen/otto.Value.export(0x5, 0x6fdb80, 0xc820121a40, 0x0, 0x0)
	/home/bas/Development/golang/src/github.com/robertkrimen/otto/value.go:694 +0xba0
github.com/robertkrimen/otto.Value.Export(0x5, 0x6fdb80, 0xc820121a40, 0x0, 0x0, 0x0, 0x0)
	/home/bas/Development/golang/src/github.com/robertkrimen/otto/value.go:648 +0x4d
main.main()
	/home/bas/Development/golang/src/github.com/bas-vk/test/main.go:11 +0xef
exit status 2
```
With `vm.Set(...)` it works as expected.

**Comment by deoxxa** [comment]
@bas-vk can you confirm that the fix in #169 works for you?

**Comment by bas-vk** [comment]
It works.
